### PR TITLE
Pull out manage pages

### PR DESCRIPTION
**Pull request by Sarahb-da**
This modifies the routes and flattens the manage tabs, moving them to the sidebar. 

There are some more hierarchical issues that I would like to address in a follow up PR, like nesting members inside clearing and so on. Once we get the hierarchy right this should provide more navigational clarity to the user...

Current structure in this PR
```
    distributions: '/app/distributions',
    instruments: {
      root: '/app/instruments',
      instrument: '/app/instruments/instrument',
      new: {
        base: '/app/instruments/new/base',
        convertiblenote: '/app/instruments/new/convertiblenote',
        binaryoption: '/app/instruments/new/binaryoption',
      },
    },
    trading: '/app/trading',
    listings: { root: '/app/listings', new: '/app/listings/new' },
    issuance: { root: '/app/issuance', new: '/app/issuance/new' },
    setup: {
      root: '/app/setup',
      identity: '/app/setup/identity',
    },
    clearing: {
      root: '/app/clearing',
      member: '/app/clearing/member',
    },
    clearingServices: {
      root: '/app/clearingServices',
      member: '/app/clearing/member',
      offer: '/app/clearing/offer',
      market: { offer: '/app/clearingServices/market/offer' },
    },
    clearingMembers: {
      root: '/app/clearingMembers',
      member: '/app/clearingMembers/member',
    },
    custody: {
      root: '/app/custody',
      offer: '/app/custody/offer',
    },
    wallet: {
      root: '/app/wallet',
      account: '/app/wallet/account',
      accounts: {
        root: '/app/wallet/accounts',
        new: '/app/wallet/accounts/new',
      },
      requests: '/app/wallet/requests',
    },
    auctions: {
      root: '/app/auctions',
      new: '/app/auctions/new',
    },
    biddingAuctions: '/app/biddingAuctions',
    markets: {
      root: '/app/markets',
      order: '/app/markets/order',
      offer: '/app/markets/offer',
    },

```

### CODE REVIEW

File: ui/src/App.tsx [hunk 1]
```diff
@@ -172,7 +160,7 @@ const AppComponent = () => {
     sidebar: [
       {
         label: 'Auctions',
-        path: paths.app.distribution.auctions,
+        path: paths.app.auctions.root,
```
Comment (Sarahb-da) [inline ui/src/App.tsx:163]: auctions are currently not nested in the distribution tab, however maybe they should be in which case we can nest the auction pages inside distribution

File: ui/src/App.tsx [hunk 2]
```diff
@@ -321,82 +304,116 @@ const AppComponent = () => {
...
+  //   },
+  //   {
+  //     path: paths.app.listings.new,
+  //     render: () => (
+  //       <ServiceRequired service={ServiceKind.LISTING} action="New Listing">
+  //         <ListingNew services={listingService} />
+  //       </ServiceRequired>
+  //     ),
+  //   },
+  // ],
```
Comment (Sarahb-da) [inline ui/src/App.tsx:399]: So I could only find Listing routes from other listing routes. Should it be in the sidebar? Or under trading? @alexgraham-da

File: ui/src/paths.ts [hunk 3]
```diff
@@ -8,63 +8,60 @@ const paths = {
...
-          base: '/app/setup/instrument/new/base',
-          convertiblenote: '/app/setup/instrument/new/convertiblenote',
-          binaryoption: '/app/setup/instrument/new/binaryoption',
-        },
-      },
-      issuance: { new: '/app/setup/issuance/new' },
-      listing: { new: '/app/setup/listing/new' },
-      trading: { offer: '/app/setup/trading/offer' },
     },
+    // TODO: nest clearing structure
```
Comment (Sarahb-da) [inline ui/src/paths.ts:28]: We separately display `clearing services`, `members`, and `member` (which is the party's member) - I think in the next PR I'll create a nested menu so all of these functions will go under `clearing`

File: ui/src/App.tsx (hunk 2 above)
Comment (alexgraham-da) [reply ui/src/App.tsx:399]: I think Listings would work in the sidebar. It's technically different than Trading in terms of the services

File: ui/src/pages/error/ServiceRequired.tsx [hunk 4]
```diff
@@ -41,7 +41,7 @@ interface RequestInterface {
 
 export const ServiceRequired: React.FC<ServiceRequiredProps> = ({ service, action, children }) => {
   const party = useParty();
-
+  console.log('here');
```
Comment (alexgraham-da) [inline ui/src/pages/error/ServiceRequired.tsx:44]: extra log

File: ui/src/pages/page/Page.tsx [hunk 5]
```diff
@@ -44,14 +44,14 @@ const Page: React.FC<Props> = ({
   const constructMenu = (sideBarItem: SidebarEntry, level: number): React.ReactElement => {
     const childMenu = sideBarItem.children.map(child => constructMenu(child, level + 1));
     const margin = level * 25;
-
+    console.log(sideBarItem.activeSubroutes);
```
Comment (alexgraham-da) [inline ui/src/pages/page/Page.tsx:47]: extra log

File: ui/src/App.tsx [hunk 6]
```diff
@@ -430,8 +447,8 @@ const AppComponent = () => {
   };
 
   const path = useLocation().pathname;
-
   const currentEntry = entriesToDisplay.find(entry => path.startsWith(getBaseSegment(entry.path)));
+  console.log(currentEntry);
```
Comment (alexgraham-da) [inline ui/src/App.tsx:451]: extra log

File: ui/src/paths.ts [hunk 7]
```diff
@@ -8,63 +8,60 @@ const paths = {
...
     clearing: {
-      members: '/app/clearing/members',
+      root: '/app/clearing',
       member: '/app/clearing/member',
     },
+    clearingServices: {
+      root: '/app/clearingServices',
```
Comment (alexmatson-da) [inline ui/src/paths.ts:34]: I think the paths would look better (in a browser address bar, for example) with kebab casing, like `/app/clearing-services`...

**Review by alexmatson-da — APPROVED** [review]
One suggestion, but lgtm!

**Pull request merged**